src/Core/Request/Orders/OrderCreateFromCartRequest.php

```diff
@@ -220,7 +265,7 @@ public function buildResponse(ResponseInterface $response)
     public function httpRequest()
     {
         $payload = [
-            static::ID => $this->getCartId(),
+            static::CART => $this->getCart(),
```

Comment:
Old code will stop working as the ID is not transmitted anymore.

So all setters/getters should set the cart field and return the ID from the cart field